src/tools/cephfs_mirror/FSMirror.h

```diff
@@ -97,18 +104,6 @@ class FSMirror {
   void reopen_logs();
 
 private:
-  bool is_blocklisted(const std::scoped_lock<ceph::mutex> &locker) const {
```

Comment:
That's a convention used to signal that the function should be called with the lock held.